Tests/IntegrationTests/Components Tests/Apple Pay/ApplePayComponentTests.swift

```diff
@@ -424,7 +424,7 @@ class ApplePayComponentTest: XCTestCase {
         )
 
         // When
-        sut.viewDidLoad(viewController: mockViewController)
+        setupRootViewController(sut.viewController)
```

Comment:
Can you try `sut.viewController.loadviewIfNeeded()` instead?
This significantly speeds up the test run as there is no real UI involved but the lifecycle remains.